pyoutline/outline/backend/cue.py

```diff
@@ -34,6 +34,8 @@
 
 import six
 
+from packaging.version import Version
```

Comment:
nit: This import can be moved above the `import six` line (grouping third-party imports together alphabetically)